downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -27,6 +27,29 @@ Update your current Ballerina installation directly to 2201.7.0 by using the [Ba
 If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
 
 ## Backward-incompatible changes
+- Fixed a bug that sets broad type for constants instead of singleton type.
+
+  ```ballerina
+  const int CONST = 1 + 7;
+  CONST num = 2;
+  ```
+  Now, the code produces an error: `incompatible types: expected '8', found 'int'`. Similarly, the following assignment in the function scope also results in an error: `incompatible types: expected 'SingletonType', found 'int'`.
+
+  ```ballerina
+  const int CONST = 3 + 5;
+  type SingletonType CONST;
+  
+  public function main() {
+    SingletonType num = 3;
+  }
+  ```
+  
+- Fixed a bug that caused a runtime error instead of a compile-time error for constant declarations containing duplicate keys in computed fields in mapping constructor expressions.
+  ```ballerina
+  const NAME = "name";
+  const map<string> person = {name : "Joe", [NAME] : "Jack"};
+  ```
+  Now, this produces an error: `invalid usage of finite literal: duplicate key 'name'`.
```

Comment:
Not necessary IMO. The actual error doesn't really matter? You can add a comment in the code like "Compilation error now."